### DISCUSSION TITLE
Adjusting the timestamp of recorded behaviours to match a specific time of day

### DISCUSSION THREAD

**Discussion by RMM84** [olivierfriard/BORIS#419] [Q&A]
Hi, 

I have video data that is relevant for a specific time of day.  I want to adjust my BORIS data to match these times. E.g. If the video was started 09:26 am then I want this time to be added to all of my times in my observations list.  When I export the data I want to have my observations list formatted in a 24 hr format and matching the actual time of day that they took place.

At present I can only seem to export the BORIS day in seconds. 

Can anyone help solve this?

**Comment by olivierfriard** [comment]
Hi RMM84,

check the Time offset parameter. See
https://boris.readthedocs.io/en/latest/#create-a-new-observation

BORIS export timestamp in seconds, you can easily write an Excel function
to convert seconds in hh:mm:ss.mss format



On Wed, Aug 4, 2021 at 10:59 AM RMM84 ***@***.***> wrote:

> Hi,
>
> I have video data that is relevant for a specific time of day. I want to
> adjust my BORIS data to match these times. E.g. If the video was started
> 09:26 am then I want this time to be added to all of my times in my
> observations list. When I export the data I want to have my observations
> list formatted in a 24 hr format and matching the actual time of day that
> they took place.
>
> At present I can only seem to export the BORIS day in seconds.
>
> Can anyone help solve this?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/olivierfriard/BORIS/discussions/419>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABR2AYVRFGXKZUQHHMZXDX3T3D6OZANCNFSM5BQTSQGA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
University of Torino
Dept of Life Sciences and Systems Biology
Via dell'Accademia Albertina, 13 10123 TORINO (Italy)
tel: +39 011 6704542
http://penelope.unito.it/friard/
http://orcid.org/0000-0002-0374-9872

**Reply by RMM84** [reply]
Hi Olivier, 

Thank you for your quick response.

So whilst it is possible to offset the time in the observations in BORIS, you can still only export these values in seconds. 

Ok thanks!
